### PR TITLE
Test github-experimental pipeline

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

**Comment by son-vyas** [comment]
recheck

**Comment by son-vyas** [comment]
check rdo experimental

**Comment by son-vyas** [comment]
check-rdo

**Comment by son-vyas** [comment]
recheck

**Comment by son-vyas** [comment]
recheck

**Comment by son-vyas** [comment]
check-rdo

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -6,3 +6,6 @@
     github-gate:
       jobs:
         - noop
+    github-experimental:
```
Comment (raukadah) [inline zuul.d/projects.yaml]:
```suggestion
    github-experimental-trigger:
```

File: zuul.d/projects.yaml [hunk 2]
```diff
@@ -6,3 +6,6 @@
...
+      jobs:
+        - create-scenario010-trigger-for-downstream-network-zed-c9-rdo-test
```
Comment (raukadah) [inline zuul.d/projects.yaml]: I would first start with a noop job. We can add the trigger job in a seperate pr.

Comment (son-vyas) [reply zuul.d/projects.yaml]: done

File: zuul.d/projects.yaml (hunk 1 above)
Comment (son-vyas) [reply zuul.d/projects.yaml]: done

**Comment by son-vyas** [comment]
experimental

File: zuul.d/test_jobs.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+- job:
```
Comment (raukadah) [inline zuul.d/test_jobs.yaml]: Please drop this file and also fix the commit message.

Comment (son-vyas) [reply zuul.d/test_jobs.yaml]: done

**Comment by son-vyas** [comment]
experimental

**Comment by raukadah** [comment]
https://softwarefactory-project.io/zuul/t/rdoproject.org/buildset/84dfeb29ec3a4092919d9390de810d9c got triggered on /test github-experimental but causing issue with prow.

**Comment by raukadah** [comment]
check-experimental

**Comment by raukadah** [comment]
check-experimental triggers https://softwarefactory-project.io/zuul/t/rdoproject.org/buildset/99145c87df9d4ed1bb3057f704f1e9e1

**Comment by raukadah** [comment]
experimental

**Comment by raukadah** [comment]
"experimental" comment trigger this https://softwarefactory-project.io/zuul/t/rdoproject.org/buildset/75bc216e25384ba0882810b0897f63e6

**Pull request merged**